SmartDeviceLink/SDLCreateWindow.h

```diff
@@ -0,0 +1,148 @@
+//
+//  SDLCreateWindow.h
+//  SmartDeviceLink
+//
+
+#import "SDLRPCRequest.h"
+#import "SDLWindowType.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/*
+ * Create a new window on the display with the specified window type.
+ * @since SDL 6.0
+ */
+@interface SDLCreateWindow : SDLRPCRequest
+
+- (instancetype)init NS_UNAVAILABLE;
```

Comment:
Because, it has required parameters.
It was made unavailable so the developer does not make incorrect initializations.

Reply:
It doesn't matter. Look at the rest of the library. We keep the standard init available, plus an initializer with required parameters, plus one with all parameters.

Reply:
The idea isn't actually bad (it's used many times in the manager layer). It can be used to avoid instances with mandatory properties being nil. If you don't agree to use this declaration in new RPCs we could discuss this in an SDLC proposal.